apps/src/applab/applab.js

```diff
@@ -161,6 +162,11 @@ function loadLevel() {
   for (var key in level.scale) {
     Applab.scale[key] = level.scale[key];
   }
+
+  if (Applab.makerlabEnabled) {
+    var BoardController = require('../makerlab/BoardController');
```

Comment:
Actually requiring it when makerlab is not enabled (and thus the browserify-shimmed dependencies are not available globally) results in:

![image](https://cloud.githubusercontent.com/assets/206973/14190224/6f459428-f746-11e5-9009-51fa93a8195d.png)